internal/usecase/client/output_view_test.go

```
@@ -0,0 +1,100 @@
+				context.Publication = 1
+			case "invalid context":
+				context.FocusedPaneID = ""
+			case "contextual side effect":
+				output.Base, output.New = 0, 0
+			case "plain side effect":
+				output.Base, output.New, output.Epoch = 0, 0, 9
+				output.Context = nil
+			}
+			next, accepted := initial.next(output)
+			require.Equal(t, scenario == "delta" || scenario == "plain side effect", accepted)
+			if scenario == "delta" {
```

golangci-lint (2.13.2)

[error] QF1003: could use tagged switch on scenario

(staticcheck)